crash-recovery/README.md

```diff
@@ -0,0 +1,78 @@
+```
+shortname: ?/CRASH-RECOVERY
+name: Restore system state after crash
+type: standard
+status: raw
+editor: Vanshdeep Singh <vanshdeep@bigchaindb.com>
+```
+
+## Abstract
+Unexpect crashes are bound to occur when BigchainDB is deployed in production which could leave the system in an inconsistant state. Hence it is necessary that an appropriate methodology should be developed to restore the system to a consistant state.
+
+
+## Motivation
+For BigchainDB, a crash which can cause the state to be inconsistent could leave the system inoperable. The 'achilles heel' for BigchainDB is its `commit` function during which a new block with its transactions is written to the disk. Furthermore, UTXO merkle tree is updated based on these new transactions. An overview of what happens during `commit` is given below,
+
+1. Store metadata of the transactions.
+1. Store assets of the transactions.
+1. Store transactions and update UTXO.
+1. Store Block.
+
+These write operations are not transactional so any intermediate failure could result in an inconsistent state. For example, 
```

Comment:
Here when you say "transactional" I think you mean in the database sense, but you should be clear about that, e.g. "_transactional_ in the database sense of the word," that is, ACID, and you can link to a reference defining ACID, maybe Wikipedia (!): https://en.wikipedia.org/wiki/ACID